### ISSUE TITLE
`uint64_t` literal `ULL` handled incorrectly

### ISSUE THREAD

**Issue by martinwinter-huawei** [o3de/o3de-azslc#49]
Using the `unsigned long long` literal `ULL` currently does not compile.
The following code:
```cpp
uint64_t test_variable = 1ULL << shift_amount;
```
will result in the following error message:
```
syntax error #1: missing ';' at 'L'
```
Using `ULL` directly in `hlsl` code works without an issue using `dxc`.

**Comment by martinwinter-huawei** [comment]
This #51 pull request fixes this issue by updating the allowed `IntegerSuffix`es of the lexer.

**Comment by delanyinspiron6400** [comment]
Properly fixed in #53

**Comment by martinwinter-huawei** [comment]
Properly fixed in #53

**Comment by martinwinter-huawei** [comment]
As this has been merged, this issue has been fixed! 👍
